mmdet3d/datasets/pipelines/compose.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import collections
+
+from mmcv.utils import build_from_cfg
+
+from mmdet.datasets.builder import PIPELINES as MMDET_PIPELINES
+from ..builder import PIPELINES
+
+
+@PIPELINES.register_module()
+class Compose:
+    """Compose multiple transforms sequentially.
```

Comment:
Is there any difference with mmdet version? May add explanations to the docstring?